src/Resources/views/Invoice/Download/pdf.html.twig

```diff
@@ -125,6 +125,17 @@
             <td>{{ '%0.2f'|format(invoice.total/100) }}</td>
             <td>{{ invoice.currencyCode }}</td>
         </tr>
+        <tr class="paid">
+            <td colspan="5"></td>
+            <td colspan="2" class="bold">{{ 'sylius_invoicing_plugin.ui.payment.paid'|trans }}:</td>
+            <td>
+                {% if invoice.paymentState() is constant('Sylius\\InvoicingPlugin\\Entity\\InvoiceInterface::PAYMENT_STATE_COMPLETED') %}
```

Comment:
I'm wondering, cannot we just use the state and display it as some kind of a badge? It's one of the benefits of using state over boolean flag 😄